libs/wallet/provider/uphold/uphold.go

```diff
@@ -1180,3 +1182,12 @@ func FundWallet(ctx context.Context, destWallet *Wallet, amount decimal.Decimal)
 
 	return balance.TotalProbi, nil
 }
+
+func redactUnneededContent(sbody string) string {
+	for _, p := range []string{`"description":\s*"[^"]+"\s*,?`, `"\w+Country":\s*"[^"]+"\s*,?`} {
```

Comment:
suggestion: I would not advocate using globals but we are already do it in that package for regex [here](https://github.com/brave-intl/bat-go/blob/2e8f6a4d9b377dc033ee5294b5c462c220428249/libs/clients/client.go#L36) and vars [here](https://github.com/brave-intl/bat-go/blob/2e8f6a4d9b377dc033ee5294b5c462c220428249/libs/wallet/provider/uphold/uphold.go#L66). Calling must which can potentially panic on every request plus the overhead seems unnecessary. In the loop we are also copying every time rather than indexing the value.